### PR TITLE
Touch-ups in Readme

### PR DESCRIPTION
**Pull request by Armaxis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -326,7 +325,7 @@ TODO: code and documentation need to be added.
 
 # License
 
-Copyright 2019 Square Inc.
+Copyright 2020 Square Inc.
```
Comment (zach-klippenstein) [inline README.md]: IANAL but if this code was written in 2019, I believe the copyright header can stay like this.

**Review by zach-klippenstein — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jrodbx) [reply README.md]: correct, copyright years only get updated for new files.

Comment (Armaxis) [reply README.md]: TIL!
Reverted back

**Comment by helios175** [comment]
Thank you sir!

**Pull request merged**